documentation/RELEASE_NOTES.md

```diff
@@ -17,6 +17,14 @@ should also be read to understand what has changed since earlier releases.
 
 <!-- Insert new items immediately below here ... -->
 
+### Support modules can now add error symbols even before calling `errlogInit`
+
+A support module can add an error symbol by calling `errSymbolAdd`, for
+example: `errSymbolAdd(S_newError, "New error message")`
+
+This used to work in older EPICS versions but got broken at some point and is
+now restored.
+
```

Comment:
I would describe this change as "Allow errSymbolAdd() to be called after early initialization".  I would also like to see some guidance to users about how avoid error number collisions.  Also, "got broken at some point" is not helpful to users.

For that matter, I think this sort of information should be added as a doxygen comment for `errSymbolAdd()`.  eg.

```c
/** Add custom error message to be returned by subsequent `errSym*` calls.
 * \since 7.0.2 Calls after errlogInit() have no effect.
 * \since UNRELEASED Can be called at any time.
 */
```

The `7.0.2` is a guess that 0fdb995a0bda9a7d9591b6c2b1bde5b2aab43421 contained the "breakage".

Reply:
I simplified the release note using the text you suggest, unfortunately I can't build your guess commit to verify it, not sure if because build was broken or because there is something nasty in my environment